feature-libs/cart/saved-cart/occ/saved-cart-occ.module.ts

```diff
@@ -0,0 +1,10 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { CostCenterOccModule, provideDefaultConfig } from '@spartacus/core';
+import { defaultOccSavedCartConfig } from './config/default-occ-saved-cart-config';
+
+@NgModule({
+  imports: [CommonModule, CostCenterOccModule],
```

Comment:
CostCenterOccModule should be removed

Reply:
that's from the base feature-lib
will remove it there